canonical-web-and-design/ubuntu.com
# Style the board and OS selection sections

## Done
Style the board and OS selection sections

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/core/build
- Check the select board and OS sections match [the design](https://app.zeplin.io/project/5e33f5075368883e5da9d0bf/screen/5e4fe76e44a1ee65293d3d7f) 

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2362

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/75170017-5f24b580-5721-11ea-96e2-87d6ba181fa9.png)
